feature.rb

```diff
@@ -0,0 +1,30 @@
+def bubble_sort(array)
+  loop do
+    moved = false
+    (array.length - 1).times do |i|
+      if array[i] > array[i + 1]
+        array[i], array[i + 1] = array[i + 1], array[i]
+        moved = true
+      end
+    end
+    break unless moved
+  end
```

Comment:
bubble sort returns a sorted array. But, based on your current formation it returns nil. after line 11, place `array`.